src/CMakeLists.txt

```diff
@@ -315,31 +320,35 @@ if (PRECOMPILE_SYSTEM_HH AND (COMMAND target_precompile_headers) AND (CMAKE_BUIL
 endif()
 
 if (USE_PYTHON)
-  if (Python_SITEARCH)
+  if (SKBUILD)
+    install(TARGETS _${SKBUILD_PROJECT_NAME} DESTINATION ${SKBUILD_PROJECT_NAME})
+  elseif (Python_SITEARCH)
     if (WIN32 AND NOT CYGWIN)
       set(_ledger_python_module_name "ledger.pyd")
     elseif(CMAKE_SYSTEM_NAME STREQUAL Darwin)
       set(_ledger_python_module_name "ledger.so")
     else()
       set(_ledger_python_module_name "ledger${CMAKE_SHARED_LIBRARY_SUFFIX}")
     endif()
-
     # FIXME: symlink would be sufficient:
     # maybe using install(CODE "...") and
     # execute_process(COMMAND "${CMAKE_COMMAND}" -E create_symlink ...).
     # Windows will need a special case due to not supporting symlinks.
     add_custom_command(
       TARGET libledger POST_BUILD
       COMMAND ${CMAKE_COMMAND} -E copy_if_different
-      $<TARGET_FILE:libledger> "${CMAKE_BINARY_DIR}/${_ledger_python_module_name}")
+      $<TARGET_FILE:libledger>
+      "${CMAKE_BINARY_DIR}/python/${SKBUILD_PROJECT_NAME}/_${_ledger_python_module_name}")
```

Comment:
ℹ️ This somewhat unusual destination is necessary in order to make the python related tests pass from CMake's build directory.